app/views/Project/Tagging/Export/NewExport/index.tsx

```diff
@@ -374,9 +422,23 @@ function NewExport(props: Props) {
         },
     );
 
+    const filteredWidgets = useMemo(
+        () => columns?.filter((w) => w.selected === true),
+        [columns],
+    );
```

Comment:
I don't think we need to `memoize` this value. This data is only used when getCreateExportData is called which is not that frequent and the `filter` operation is a fast operation and doesn't involve any additional calculation overhead.